tox.ini

```diff
@@ -102,6 +94,19 @@ commands_pre = rm -rf dist/
 changedir = tests/non-pytest/poetry-lock-test
 commands = poetry lock
 
+[testenv:freezedeps]
+# do *not* use pip-compile-multi for the mindeps ones because it conflicts with what
+# we're doing with the mindeps environments
+description = freeze development dependencies using pip-compile and pip-compile-multi
+skip_install = true
+deps =
+    pip-tools
+    pip-compile-multi
+commands =
+    pip-compile-multi
+    pip-compile -q --resolver=backtracking {toxinidir}/requirements/test-mindeps.isolated -o {toxinidir}/requirements/test-mindeps.txt
+    pip-compile -q --resolver=backtracking {toxinidir}/requirements/typing-mindeps.isolated -o {toxinidir}/requirements/typing-mindeps.txt
```

Comment:
This is where the absolute paths are getting introduced. Hopefully it's possible to strip the leading `{toxinidir}/` from this command and rely on the current working directory! :crossed_fingers:

Reply:
Nice catch!
I hadn't even considered that tox would expand that and that would be the cause of the problem.